src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_addresses.html.twig

```diff
@@ -1,17 +1,21 @@
-<h4 class="ui top attached styled header">
-    {{ 'sylius.ui.shipping_address'|trans }}
-</h4>
-<div class="ui attached segment" id="shipping-address">
-    {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.shippingAddress} %}
-</div>
-<h4 class="ui attached styled header">
-    {{ 'sylius.ui.billing_address'|trans }}
-</h4>
-<div class="ui attached segment" id="billing-address">
-    {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.billingAddress} %}
-    <div class="ui segment">
-        <a href="{{ path('sylius_admin_order_update', {'id': order.id}) }}" class="ui icon labeled tiny fluid button">
-            <i class="pencil icon"></i> {{ 'sylius.ui.edit'|trans }}
-        </a>
+{% if order.shippingAddress is not null %}
+    <h4 class="ui top attached styled header">
+        {{ 'sylius.ui.shipping_address'|trans }}
+    </h4>
+    <div class="ui attached segment" id="shipping-address">
+        {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.shippingAddress} %}
+    </div>
+{% endif %}
+{% if order.billingAddress is not null %}
+    <h4 class="ui attached styled header{% if order.shippingAddress is null %} top{% endif %}">
+        {{ 'sylius.ui.billing_address'|trans }}
+    </h4>
+    <div class="ui attached segment" id="billing-address">
+        {% include '@SyliusAdmin/Common/_address.html.twig' with {'address': order.billingAddress} %}
     </div>
+{% endif %}
+<div class="ui attached segment" id="edit-addresses">
```

Comment:
Shouldn't we hide it when no addresses are set?

Reply:
I was wondering about that, but I can imagine that SOMEHOW addresses could not be set by mistake... and then we want to be able to set them in the Admin panel, I suppose 😄